src/scripts/main.js

```diff
@@ -1,5 +1,27 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const wallHight = wall.clientHeight;
+const wallWidth = wall.clientWidth;
+const spiderHight = spider.clientHeight;
```

Comment:
The variable name `spiderHight` is misspelled. It should be `spiderHeight` to accurately reflect its purpose.